install/scripts/aws-up.sh

```diff
@@ -0,0 +1,93 @@
+#!/bin/bash
+
+set -e
+set -u
+
+# Required env variables:
+: "$AWS_ACCOUNT_ID $AWS_ACCESS_KEY_ID $AWS_SECRET_ACCESS_KEY"
+
+INSTALL_OPERATOR="${INSTALL_OPERATOR:-yes}"
+
+SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+KUBERENTES_DIR=${SCRIPT_DIR}/../kubernetes
+
+EKSCTL_ENABLE_CREDENTIAL_CACHE=1
+export CLUSTER_NAME=substratus
+export REGION=us-west-2
+export ARTIFACTS_REPO_NAME=${CLUSTER_NAME}
+export ARTIFACTS_BUCKET_NAME=${AWS_ACCOUNT_ID}-${CLUSTER_NAME}-artifacts
+export KARPENTER_VERSION=v0.29.2
+export AWS_PARTITION="aws"
```

Comment:
If including partition, lets add a link to what it is: https://docs.aws.amazon.com/whitepapers/latest/aws-fault-isolation-boundaries/partitions.html

I am not sure we need this for our purposes.

Reply:
I'm comfortable hardcoding it and not documenting

From: https://karpenter.sh/v0.29/getting-started/getting-started-with-karpenter/#2-set-environment-variables